.env.test

```diff
@@ -29,3 +29,6 @@ FEED_DEV_INFRA_CHANNEL_ID="13141516"
 DISABLE_GITHUB_METRICS="false"
 FORCE_USER_TOKEN_GITHUB_CLIENT=
 FORCE_GET_USER_BY_SLACK_ID=
+TEAM_OSPO_CHANNEL_ID=
```

Comment:
What do this line and the one below it mean? Are they related to DRY_RUN, or separate things that are being added to avoid a whole separate PR? If the latter, it might be useful to add a line in the commit body that you've also added this orthogonal stuff to the example env.

Reply:
Yeah, they're separate things I noticed to avoid a whole separate PR since it's touching on the same thing. I'll add to commit body